cmd/booster-http/gateway_handler.go

```diff
@@ -14,19 +14,26 @@ type gatewayHandler struct {
 	supportedFormats map[string]struct{}
 }
 
-func newGatewayHandler(gw *gateway.BlocksBackend, supportedFormats []string) http.Handler {
+func newGatewayHandler(gw *gateway.BlocksBackend, supportedFormats []string, serveGateway string) http.Handler {
```

Comment:
I don't believe we need the supportFormats parameter any more.
I think we can also get rid of most of the code in gateway_handler.go

Reply:
ack. changes made on the latest commit.